src/app/accounts/active-accounts.ts

```diff
@@ -0,0 +1,38 @@
+import { MS_PER_30_DAYS, USER_ACTIVENESS_MONTHLY_VOLUME_THRESHOLD } from "@config/app"
+import { LedgerService } from "@services/ledger"
+import { AccountsRepository } from "@services/mongoose"
+
+// user is considered active if there has been one transaction of more than USER_ACTIVENESS_MONTHLY_VOLUME_THRESHOLD sats in the last 30 days
+const isAccountRecentlyActive = async (walletId: WalletId) => {
```

Comment:
there is a naming inconsistency here `isAccount...` but argument `walletId` - why?
In the desired future state are we measuring activity of the account or the user or the wallet?

Reply:
we're measuring the activity of the wallets attached to an account. I've done a refactoring to make it future proof when we'll have multiple wallet per account (also this assume account are denominated BTC currently which is still la shortcut)